run_simulations.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+import subprocess
+
+def check_directory(dirname : str):
+    if not os.path.isdir(dirname):
+        os.mkdir(dirname)
+
+def main(args):
+    assert(len(args) >= 4)
+    simulator_path = args[1]
+    simulation_configs_dif = args[2]
+    corner_metrics_dir = args[3]
+
+    check_directory(corner_metrics_dir)
+
+    for filename in os.listdir(simulation_configs_dif):
+        if not filename.endswith('.yml'):
```

Comment:
Maybe add `yaml` extension?
```suggestion
        if not filename.endswith('.yml') and not filename.endswith('.yaml'):
```